task-def-api.json

```
@@ -21,6 +21,10 @@
     "linuxParameters": null,
     "cpu": ${cpu},
     "environment": [
+      {
+        "name": "API_BASE_URL",
+        "value": "${API_BASE_URL}"
```

I think this should work. You could then remove the references to `api_base_url` in the other files.
```suggestion
        "value": "${HOST}"
```

---

:+1: